src/index.html

```diff
@@ -11,7 +11,180 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <div class="form-personal">
+          <label for="surname">Surname:</label>
+          <input
+            id="surname"
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="form-personal">
+          <label for="name">Name:</label>
+          <input
+            id="name"
+            type="text"
+            name="name"
+            autocomplete="off"
+            required
+          >
+        </div>
+
+        <div class="form-personal">
+          <label for="age">How old are You?</label>
+          <input
+            id="age"
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </div>
+
+        <div class="form-personal">
+          <label for="birthday">Full date of birth:</label>
+          <input
+            id="birthday"
+            type="date"
+            name="birthday"
+          >
+        </div>
+
+        <div class="form-personal">
+          <label for="accept">I accept the term of the agreement</label>
+          <input
+            id="accept"
+            type="checkbox"
+            name="accept"
+            required
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <div class="form-personal">
+          <label for="mail">E-mail:</label>
+          <input
+            id="mail"
+            type="email"
+            name="mail"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+
+        <div class="form-personal">
+          <label for="password">Password:</label>
+          <input
+            id="password"
+            type="password"
+            name="password"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-personal">
+          <label>Do you love cats?
```

Comment:
check list p2, fix in all cases

```suggestion
          <label>
          Do you love cats?
```